app/models/user.rb

```diff
@@ -11,4 +11,6 @@ class User < ActiveRecord::Base
 
   has_many :invitations, class_name: 'Invitation', foreign_key: 'recipient_id'
   has_many :sent_invitations, class_name: 'Invitation', foreign_key: 'sender_id'
+
+  attr_accessor :token
```

Comment:
napewno da sie tak napisac kod zeby nie uzywac attr_accesora ale to juz na korkach pomyślimy